contracts/pallet-ibc/src/ics20_fee/mod.rs

```diff
@@ -0,0 +1,261 @@
+// Copyright 2022 ComposableFi
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We need to remove APACHE

Reply:
To be replaced with what? It's still Apache used all over centauri

Reply:
we need to remove it everywhere
@KaiserKarel

Reply:
Yeah Apache needs to be removed everywhere in the codebase until we launch and decide on a final license with legal.